docs/styles.md

```diff
@@ -797,7 +797,10 @@ The Accordion component is designed to showcase content in an expandable and col
 **Example**
 
 <div style="text-align: center;">
-    <img src="./static/accordion.png" width="600">
+    <img src="./static/accordion_1.png" width="600">
```

Comment:
Thanks for spinning this up, @mjoss17 !

I think right now the images don't work due to a mismatch between the PNG file names and what is hear (e.g. `accordion_1.png` vs `accordian_1.png`).

Reply:
I went ahead and made this fix real quick so we can get this merged.